Code/Framework/AzCore/AzCore/Math/Aabb.h

```diff
@@ -43,10 +43,10 @@ namespace AZ
         static Aabb CreateCenterRadius(const Vector3& center, float radius);
 
         //! Creates an AABB which contains the specified points.
-        static Aabb CreatePoints(const Vector3* pts, size_t numPts);
+        static Aabb CreatePoints(const Vector3* points, size_t pointCount);
 
         //! Creates an AABB which contains the specified points.
-        static Aabb CreatePoints(const AZStd::span<Vector3>& points);
+        static Aabb CreatePoints(const AZStd::span<const Vector3> points);
```

Comment:
nit: Please remove the left `const` here on the interface, it is not needed (goes for all pass by value parameters, it can be useful to still mark them `const` in the implementation though, I personally like doing this to show the parameter doesn't change)